### ISSUE TITLE
Implement functionality for login/signup pages

### ISSUE THREAD

**Issue by ahmetfirat23** [bounswe/bounswe2024group12#122]
## Description

This task includes the implementation of functionality for login and signup pages. The buttons and hyperlinks on the page must be functional. User input should be checked according to requirements. Assuming an endpoint, login and signup mechanisms should be implemented.

***

### Reviewer:
This issue will be reviewed by: _@asyasusen_

**Comment by sonerkuyar** [comment]
I implemented some functionalities and Page routing to the SignUp and Login pages. Please check the App which are implemented which are not.

**Comment by ahmetfirat23** [comment]
> I implemented some functionalities and Page routing to the SignUp and Login pages. Please check the App which are implemented which are not.

Great, I added sha256 encryption decryption to user input. Also made a temporary home page to access this pages without manually changing the link. Currently I am working on checking username and password match requirements criteria and I will show a message if user inputs something wrong.

**Comment by ahmetfirat23** [comment]
I added those features too. Also added a page for successful signup that says check your mailbox. I was thinking of doing it as a pop-up but this way seemed more reasonable. Also I am passing the success with navigate function so users cannot just enter url and access that page. Finally, I included a card for invalid accesses like this one, it has a button to navigate the main page. I am thinking of preparing the connections for endpoints and will open another issue for that if that's okey with you @asyasusen @sonerkuyar .
